New version: RationalFunctionApproximation v0.1.4

JuliaRegistrator opened the pull request:
- Registering package: RationalFunctionApproximation
- Repository: https://github.com/complexvariables/RationalFunctionApproximation.jl
- Created by: @tobydriscoll
- Version: v0.1.4
- Commit: e8d33f12416354e6f66a7ad22028de3690923f38
- Reviewed by: @tobydriscoll
- Reference: https://github.com/complexvariables/RationalFunctionApproximation.jl/commit/e8d33f12416354e6f66a7ad22028de3690923f38#commitcomment-145094908
- Description: Julia package for approximation by rational functions
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
- Updated for v0.3 of `ComplexRegions`. You can now specify a different floating point precision through the domain specification, as well as by an explicit keyword argument.
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6141e7cc9c9150acf65c98b67930873337db027265909fb5a66e02ed6d722896cb6ae3d02884e908fe800d8fb7280107c7ae4cafcae17fe30291a1069fed08c02b88f3c18514264955eadc99e96d1c0ea0bf750dbf79ef877c578f9e97918806472fa8dfb81dcf2d8dca1eda94a60a0912a43a44cefd6cc3ecfa0d6e2ec5dd2575c0e1acd8d3ce8d8cd5cc6ae1c4724cd71613dc856453d456f3774b533b3fd43775f1d26ff2663634d803fedb748f9ca9176370bd1049781f555224f97e0c0c67dfc4ae6bebd395a310596a366616c87 -->

tobydriscoll commented:
looks like I never registered v0.1.3 properly, so I will stop this one and re-initiate a PR for v0.1.3

JuliaRegistrator committed:
New version: RationalFunctionApproximation v0.1.4

UUID: c92886a3-cb30-4e18-b84f-372dde475ecc
Repo: https://github.com/complexvariables/RationalFunctionApproximation.jl.git
Tree: 63ab7fd5668587407ef7a68d0271af586da5c33d

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

tobydriscoll commented:
[noblock]
